app/views/hq/orphans/show.html.erb

```diff
@@ -0,0 +1,230 @@
+    <dl class="dl-horizontal">
+      <dt>Father Cause of Death</dt>
+      <dd><%= @orphan.father_cause_of_death %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Father Date of Death</dt>
+      <dd><%= format_full_date @orphan.father_date_of_death %></dd>
+    </dl>
+  </div>
+</div>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">
+      Guardian Details
+    </h3>
+  </div>
+  <div class="panel-body">
+    <dl class="dl-horizontal">
+      <dt>Guardian Name</dt>
+      <dd><%= @orphan.guardian_name %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Guardian Relationship</dt>
+      <dd><%= @orphan.guardian_relationship %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Guardian Id Num</dt>
+      <dd><%= @orphan.guardian_id_num %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Guardian Contact Number</dt>
+      <dd><%= @orphan.contact_number %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Alt Contact Number</dt>
+      <dd><%= @orphan.alt_contact_number %></dd>
+    </dl>
+  </div>
+</div>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">
+      Original Address
+    </h3>
+  </div>
+  <div class="panel-body">
+    <dl class="dl-horizontal">
+      <dt>Province Name</dt>
+      <dd><%= @orphan.original_address.province.name %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>City</dt>
+      <dd><%= @orphan.original_address.city %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Neighborhood</dt>
+      <dd><%= @orphan.original_address.neighborhood %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Street</dt>
+      <dd><%= @orphan.original_address.street %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Details</dt>
+      <dd><%= @orphan.original_address.details %></dd>
+    </dl>
+  </div>
+</div>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">
+      Current Address
+    </h3>
+  </div>
+  <div class="panel-body">
+    <dl class="dl-horizontal">
+      <dt>Province Name</dt>
+      <dd><%= @orphan.current_address.province.name %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>City</dt>
+      <dd><%= @orphan.current_address.city %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Neighborhood</dt>
+      <dd><%= @orphan.current_address.neighborhood %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Street</dt>
+      <dd><%= @orphan.current_address.street %></dd>
+    </dl>
+    <dl class="dl-horizontal">
+      <dt>Details</dt>
+      <dd><%= @orphan.current_address.details %></dd>
+    </dl>
+  </div>
+</div>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">
+      Additional Details
+    </h3>
+  </div>
+  <div class="panel-body">
+    <dl class="dl-horizontal">
+      <dt>Spnsr Another Organization</dt>
```

Comment:
Is "Spnsr" deliberate? If space is a concern, can we do "Sponsored by Another Org." instead? Or "Sponsored by Third Party"?